webrender/src/tiling.rs

```diff
@@ -129,7 +129,10 @@ impl AlphaBatchHelpers for PrimitiveStore {
                     BlendMode::Alpha
                 }
             }
-            PrimitiveKind::Image => {
+            PrimitiveKind::Image |
+            PrimitiveKind::AlignedGradient |
+            PrimitiveKind::AngleGradient |
+            PrimitiveKind::RadialGradient => {
```

Comment:
given that you added all the primitive kinds, let's remove the `_` case at the end?